distribution/docs/src/main/resources/content/_tables/org.codice.ddf.catalog.ui.config-table-contents.adoc

```diff
@@ -235,5 +235,12 @@ TMS example (3d map support only): `[{"type" : "TMS", "url" : "https://cesiumjs.
 |
 |true
 
+|Maximum Endpoint Upload Size
+|maximumUploadSize
+|Integer
+|The maximum size of data that the Intrigue REST endpoints will receive. Please note this is not related to the product upload size, this only refers to the json data sent and received from the Intrigue endpoints.
```

Comment:
❓ Are the reasons we would suggest modifying this value or should it typically kept the same. Not sure if there are any specific conditions that should be considered.

Reply:
When a request that is too large comes through we [report an error code to the user](https://github.com/codice/ddf/blob/master/catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/util/EndpointUtil.java#L500-L502) on the front end and leave a corresponding [detailed log message for the administrator](https://github.com/codice/ddf/blob/master/catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/util/EndpointUtil.java#L487-L490) that has instructions that this value can be modified to accommodate larger requests if they are valid.

So generally the administrator would be directed here if/when it is needed.

Reply:
We can certainly add more specifically here in the docs if you think it could use it.